test/integration/install.test.js

```diff
@@ -45,12 +40,6 @@ function stripNodeBuiltIn(stdout) {
 function stripStacktrace(stdout) {
   return stdout.replace(/^\s+at\s+.*/gm, ''); // this is OK to show to the user but annoying to have in a test
 }
-function stripAnsiEscapes(stdout) {
-  return stdout.replace(
-    /[\u001B\u009B][[\]()#;?]*(?:(?:(?:[a-zA-Z\d]*(?:;[-a-zA-Z\d\/#&.:=?%@~_]*)*)?\u0007)|(?:(?:\d{1,4}(?:;\d{0,4})*)?[\dA-PR-TZcf-ntqry=><~]))/g,
-    '',
-  );
-}
```

Comment:
We actually handle this a little better now with the `NO_COLOR` env var in our test command (`"test": "NO_COLOR=true jest …"`)